miio/viomivacuum.py

```diff
@@ -201,17 +268,24 @@ def is_on(self) -> bool:
         return self.state in cleaning_states
 
     @property
-    def mode(self):
-        """Active mode.
+    def edge_state(self) -> ViomiEdgeState:
+        """Vaccum along the edges
 
-        TODO: is this same as mop_type property?
+        The settings is valid once
+        0: disabled
+        2: enabled
+        5: unknown
```

Comment:
These values are different than what is defined in `ViomiEdgeState`.

Reply:
fixed